src/cache/diagnostic_edmf_precomputed_quantities.jl

```diff
@@ -706,6 +794,7 @@ NVTX.@annotate function set_diagnostic_edmf_precomputed_quantities_top_bc!(
     n = n_mass_flux_subdomains(p.atmos.turbconv_model)
     (; ᶜentrʲs, ᶜdetrʲs) = p.precomputed
     (; ᶠu³⁰, ᶠu³ʲs, ᶜuʲs, ᶠnh_pressure³ʲs) = p.precomputed
+    precip_model = p.atmos.precip_model
```

Comment:
```suggestion
    (; precip_model) = p.atmos
```
(Just to be consistent with other parts of the code)